phpBB/language/en/common.php

```diff
@@ -110,6 +110,8 @@
 	'AVATAR_NOT_UPLOADED'			=> 'Avatar could not be uploaded.',
 	'AVATAR_NO_TEMP_DIR'			=> 'Temporary folder could not be found or is not writable.',
 	'AVATAR_NO_SIZE'				=> 'The width or height of the linked avatar could not be determined. Please enter them manually.',
+	'AVATAR_NO_UPLOAD_DIR'			=> 'Avatar storage path does not exist or is not writable.',
+	'AVATAR_NO_UPLOAD_PATH'			=> 'Avatar uploading is enabled but avatar storage path is not set.',
```

Comment:
* No storage path available, either it does not exists or is not writeable.
* Avatar storage path not set, although avatar upload is enabled.

Just a suggestion, hopefully some mother tongue will chime on this.

Reply:
I think the currently proposed wording is fine and in line with already existing language variables.